### ISSUE TITLE
extend retrospective for squid plot?

### ISSUE THREAD

**Issue by taylori** [cgrandin/hake-assessment#357]
The early years of the 1999 cohort are dropping away from the squid plots now that the cohort is older than the 15 years of the retro. It's impressive to me that the model even holds together with 15 years of data removed, but should we consider extending the retrospective to go for longer, perhaps whatever length would be necessary to see the 1999 at age 1 or age 0?

**Comment by aaronmberger** [comment]
I was wondering the same thing - is it time to omit the 99 year class?  new squid plot isn't going to show anything different for 99.  If we include it, do we just change the retrospective setup to 1:16 instead of 1:15?

**Comment by taylori** [comment]
The **easiest** option would be to just leave it as it is, and maybe bring it up at the SRG to see if they have input on the value of continuing to make comparisons to the 1999 cohort here and elsewhere. Although I guess it's not hard to re-run build with run.retro=TRUE.

**Comment by cgrandin** [comment]
My recollection was that we used the 1999 cohort as an example in the last few years when describing these plots. The trend was clear and comparable to say the 2010 cohort.

Aaron, yes that's all we would have to change and Ian is right about the call to build().

**Comment by aaronmberger** [comment]
I guess it does provide a solid example of how long it can take to stabilize a rec dev (near 10 years for this year class).  I change my vote to keep it - feeling flip floppy today

**Comment by andrew-edwards** [comment]
Yes, agree to keep it as it would be good to compare it to the 2010 value.

**Comment by andrew-edwards** [comment]
Squid plots have some numbers hard-wired. I re-ran the retros after Aaron changed the number of years, but the figs and captions aren't fully automated. I'll do that now (so we can see the new figures).

**Comment by andrew-edwards** [comment]
Updated figure setting in f77382ef. Was going to automate but seems like we'd choose every year anyway what to show, so it's still hardwired (just don't tell Chris).

**Comment by taylori** [comment]
Reopening this issue to note that I just made a change in be99be5 that I think will be useful for the final document. That is to include the 2016 cohort in the set shown in the squid plot by changing retro.cohorts from 1999:2015 to 1999:(end.yr-2).

While working on issue #410 I realized that displaying 2016 is useful to see how it compares to other big cohorts. It appears to be only slightly below where 2010 and 2014 were at this point.

No reason to leave this issue open unless someone has an objection to the change.
